db/dcrpg/sync.go

```diff
@@ -159,6 +160,30 @@ func (pgb *ChainDB) SyncChainDB(ctx context.Context, client rpcutils.MasterBlock
 		return lastBlock, fmt.Errorf("IBDComplete failed: %v", err)
 	}
 
+	// Check and report heights of the DBs. dbHeight is the lowest of the
+	// heights, and may be -1 with an empty DB.
+	stakeDBHeight := int64(pgb.stakeDB.Height())
+	if lastBlock < -1 {
+		panic("invalid starting height")
+	}
+
+	log.Info("Current best block (chain server):    ", lastBlock)
```

Comment:
This one is not `(chain server)`, but the postgresql db height, whatever we want to call that.
But also logging `nodeHeight` around here would be good too.